### PR TITLE
Document reporting endpoints based on reporter type

### PR DESCRIPTION
**Pull request by pavolloffay**

### CODE REVIEW

File: config.proto [hunk 1]
```diff
@@ -35,7 +35,9 @@ message AgentConfig {
 // Reporting covers the options related to the mechanics for sending data to the
 // tracing server o collector.
 message Reporting {
-    // endpoint represents the endpoint for reporting the traces e.g. http://api.traceable.ai:9411/api/v2/spans
+    // endpoint represents the endpoint for reporting the traces
+    // For ZIPKIN reporter type use http://api.traceable.ai:9411/api/v2/spans
```
Comment (davexroth) [inline config.proto:39]: these should be https if we're use the traceable.ai as an example.

Also, for OTLP, isn't it grpc - in which case do we need the http:// ?

Comment (jcchavezs) [inline config.proto:39]:
```suggestion
    // For Zipkin reporter type use http://api.traceable.ai:9411/api/v2/spans
```

Comment (pavolloffay) [reply config.proto:39]: `ZIPKIN` is the exact type name.

Comment (pavolloffay) [reply config.proto:39]: It worked for gRPC. See https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/protocol/exporter.md#configuration-options

Comment (davexroth) [reply config.proto:39]: ah, that's cool, I hadn't realized they've normalized that.

**Review by jcchavezs — APPROVED** [review]

**Review by mohit-a21 — APPROVED** [review]

**Pull request merged**